schema/filter_artworks.js

```diff
@@ -24,6 +28,20 @@ export const FilterArtworksType = new GraphQLObjectType({
       description: 'Artwork results.',
       type: new GraphQLList(Artwork.type),
     },
+    artworks_connection: {
+      type: artworkConnection,
+      args: pageable(),
+      resolve: ({ hits, aggregations }, options) => {
+        if ((!aggregations && !aggregations.total)) {
+          throw new Error('This query must contain the total aggregation');
```

Comment:
I really have a hard time parsing this, isn’t this essentially doing the following?

```js
if (aggregations === null && aggregations.total === null) {
}
```

Which, when reduced more, equals:

```js
if (null && null.total === null) {
}
```

Which means the second conditional is never going to be evaluated.